install/azure.go

```diff
@@ -123,7 +123,7 @@ func (k *K8sInstaller) createAzureServicePrincipal(ctx context.Context) error {
 	}
 
 	k.Log("✅ Derived Azure node resource group %s from resource group %s", clusterInfo.NodeResourceGroup, k.params.Azure.ResourceGroupName)
-	k.params.Azure.ResourceGroup = clusterInfo.NodeResourceGroup
+	k.params.Azure.NodeResourceGroup = clusterInfo.NodeResourceGroup
```

Comment:
So the commit message says (bold is mine):

> Rename to avoid confusion between the **user-given resource group** and the automatically created **AKS resource group** contained within the former.

And the new name is *node resource group*. I'm now very confused :sweat_smile: Is node resource group the user-given resource group or the AKS resource group?

Reply:
The AKS one ;) It seemed obvious to me considering:

- The existing code and contrast between the RG passed as parameter and the one derived later on.
- The Azure API and its objects (the `az aks show` command pertains to the AKS subcommand group, not the RG subcommand group -- which is `az group`).
- The nodes are entirely part of the AKS cluster, the node resource group is the RG automatically created by AKS to hold the nodes.

I can the change the commit / code to make it clearer to non-AKS persons if need be.

Reply:
Fixed in latest push.